examples/runtime/starlark/using-path/README.md

```diff
@@ -0,0 +1,21 @@
+# Starlark Path Example
+
+In this example, starlark source is provided using path in the function
+configuration.
+
+## Function invocation
+
+Get the config example and try it out by running the following commands:
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/runtime/starlark/using-path .
+kpt fn run --mount type=bind,src="$(pwd)/using-path/star-source",dst=/starlark/ using-path
```

Comment:
we should avoid using mount. Otherwise, how does this work for declarative functions?

Reply:
Do you mean that we don't advocate using mount and URL by remove these 2 examples? Or we should remove the capability from the function?
If the latter, `StarlarkFunction` CRD can be simplified to
```
apiVersion
kind
metadata
source string
data map[string]string
```

Reply:
We definitely should support running remote scripts so let's remove that from the function.

It can be useful to avoid wrapping a script in KRM, but note:
1) Gatekeeper constraints already do this, so there's a precedence
2) We can shift the problem left and providing tooling to automate wrapping the script in KRM. The user shouldn't have to author KRM as that's a poor UX.

So don't advertise mount through examples for now.

Reply:
* should NOT support remote scripts